style.css

```diff
@@ -0,0 +1,199 @@
+nav, footer {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    background-color: rgb(61, 66, 52);
+    height: fit-content;
+    width: 100vw;
+    padding-top: 10px;
+}
+
+nav h1 {
+    padding-left: 10px;
+}
+
+.links {
+    padding-right: 10px;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: flex-end;
+    align-items: center;
+    width: 50%;
+}
+
+.nav-links {
+    text-decoration: none;
+    color: antiquewhite;
+    :hover {
+        color: rgb(35, 32, 32);
+        font-style: italic;
+    }
+    padding-right: 10px;
+    margin-right: 10px;
+}
+
+main {
+    background-color: rgb(35, 32, 32);
+    color: antiquewhite;
+    min-height: 100vh;
+    height: fit-content;
+}
+
+.subheader {
+    text-align: center;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    /* border-bottom: solid;
+    border-width: 8px; */
+}
+
+.main {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: left;
+    width: 45%;
+    height: fit-content;
+    padding-bottom: 15px;
+    /* border-style: solid; */
+}
+
+.main-portfolio {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: left;
+    width: 500px;
+    height: fit-content;
+    padding: 15px;
+    border-style: solid;
```

Comment:
Class attribute names are clear and constructed. It is clear to me where each respective selector is being used.